main.py

```diff
@@ -131,6 +131,61 @@ def calculate_interest(self, rate: float) -> None:
         print("Debit accounts do not earn interest.")
 
 
+class SavingsAccount:
+    def __init__(self, account_number: str, balance: float = 0) -> None:
```

Comment:
I see no new functionalities here, compared to `Account`'s initializer. It could be safely removed after introducing inheritance.